hydra-node/src/Hydra/HeadLogic.hs

```diff
@@ -506,15 +575,30 @@ onOpenNetworkAckSn Environment{party} openState otherParty snapshotSignature sn
   requireValidAckSn $ do
     -- Spec: wait ŝ = s
     waitOnSeenSnapshot $ \snapshot sigs -> do
-      -- Spec: (j,.) ∉ ̂Σ
+      -- Spec: require (j,⋅) ∉ Σ̂
       requireNotSignedYet sigs $ do
+        -- Spec Gap: missing Σ̂[j] ← σⱼ
+        -- Spec: if ∀k ∈ [1..n] : (k,·) ∈ Σ̂
         ifAllMembersHaveSigned snapshot sigs $ \sigs' -> do
-          -- Spec: σ̃ ← MS-ASig(k_H, ̂Σ̂)
+          -- Spec: σ̃ ← MS-ASig(kₕˢᵉᵗᵘᵖ,Σ̂)
           let multisig = aggregateInOrder sigs' parties
+          -- Spec Gap: missing
+          --         η ← combine(𝑈ˆ)
+          --         ηω ← combine(outputs(txω ))
+          -- Spec: require MS-Verify(k ̃H, (cid‖v^‖ŝ‖η‖ηω), σ̃)
```

Comment:
There is a difference here: In the spec we verify using $tx_\omega$ while in the code we use the `seenSnapshot` (resolved in `waitOnSeenSnapshot` into `snapshot`). Don't think this is a problem, but it could be cleare if we indeed do $\hat{\mathcal S} \gets \mathsf{snObj}(...)$ in the spec.